runtimeconfig/google/cloud/runtimeconfig/config.py

```diff
@@ -0,0 +1,260 @@
+# Copyright 2016 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Create / interact with Google Cloud RuntimeConfig configs."""
+
+from google.cloud.exceptions import NotFound
+from google.cloud.runtimeconfig._helpers import config_name_from_full_name
+from google.cloud.runtimeconfig._helpers import variable_name_from_full_name
+from google.cloud.runtimeconfig.variable import Variable
+
+
+class Config(object):
+    """A Config resource in the Cloud RuntimeConfig service.
+
+    This consists of metadata and a hierarchy of variables.
+
+    See:
+    https://cloud.google.com/deployment-manager/runtime-configurator/reference/rest/v1beta1/projects.configs
+
+    :type client: :class:`google.cloud.runtimeconfig.client.Client`
+    :param client: A client which holds credentials and project configuration
+                   for the config (which requires a project).
+
+    :type name: string
+    :param name: The name of the config.
+    """
+
+    def __init__(self, client, name):
+        self._client = client
+        self.name = name
+        self._properties = {}
+
+    def __repr__(self):
+        return '<Config: %s>' % (self.name,)
```

Comment:
@jonparrott We have been using `%` throughout the `google-cloud-python` codebase:  I'd say leave as-is, and deal with a conversion to `format()` as a separate (and debatable) issue.